app/assets/stylesheets/styles/components/crops/components/tags-input.scss

```diff
@@ -0,0 +1,391 @@
+      border-color: $gray;
+      background: darken($gray-bg, 1%);
+      outline: none;
+
+      .tag-item {
+        background: darken($light-gray, 10%);
+      }
+    }
+  }
+
+  .autocomplete {
+    border-radius: 0;
+    background-color: darken($gray-bg, 1%);
+
+    .suggestion-item {
+      &.selected {
+        color: darken($off-black, 10%);
+        background-color: whitesmoke;
+
+        em {
+          color: darken($off-black, 10%);
+          background-color: whitesmoke;
+        }
+      }
+
+      em {
+        color: black;
+        background-color: $white;
+      }
+    }
+  }
+
+  &.ng-invalid {
+
+    .tags {
+      border-color: $of-red;
+      box-shadow: inset 0 1px 1px $background-color, 0 0 6px $of-red;
+    }
+  }
+
+  &.ti-input-lg {
+
+    .input-group-lg {
+
+      .keywords {
+        min-height: 46px;
+      }
+    }
+
+    .tags {
+
+      .input-group-lg {
+
+        .keywords {
+
+          .tags {
+            border-radius: 0;
+          }
+        }
+      }
+
+      .tag-item {
+
+        .input-group-lg {
+
+          .keywords {
+
+            .tags {
+
+              .tag-item {
+                height: 38px;
+                line-height: 37px;
+                font-size: 18px;
+                border-radius: 0;
+              }
+            }
+          }
+        }
+
+        .remove-button {
+
+          .input-group-lg {
+
+            .keywords {
+
+              .tags {
+
+                .tag-item {
+
+                  .remove-button {
+                    font-size: 20;
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+
+      .input {
+
+        .input-group-lg {
+
+          .keywords {
+
+            .tags {
+
+              .input {
+                height: 38px;
+                font-size: 18px;
+              }
+            }
+          }
+        }
+      }
+    }
+
+    &.ti-input-sm {
+
+      .input-group-sm {
+
+        .keywords {
+          min-height: 30;
+        }
+      }
+
+      .tags {
+
+        .input-group-sm {
+
+          .keywords {
+
+            .tags {
+              border-radius: 0;
+            }
+          }
+        }
+
+        .tag-item {
+
+          .input-group-sm {
+
+            .keywords {
+
+              .tags {
+
+                .tag-item {
+                  height: 22px;
+                  line-height: 21px;
+                  font-size: 12px;
+                  border-radius: 0;
+                }
+              }
+            }
+          }
+
+          .remove-button {
+
+            .input-group-sm {
+
+              .keywords {
+
+                .tags {
+
+                  .tag-item {
+
+                    .remove-item {
+
+                      font-size: 16px;
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+
+        .input {
+
+          .input-group-sm {
+
+            .keywords {
+
+              .tags {
```

Comment:
I can see why hound is complaining about this, it is a lot of nesting. Can you think of a way of refactoring this so that it's not so deep? Maybe you don't need to every layer? 

Eg. `.tag-item .tags { }` might be enough?